### DISCUSSION TITLE
Soroban Social Network

### DISCUSSION THREAD

**Discussion by snowstorm134** [stellar/sorobounty-spectacular#28] [General]
Soroban Social Network - social network built on Soroban Futurenet

Codebase: https://github.com/snowstorm134/SorobanSocialNetwork
Tutorial: https://dev.to/user1122/soroban-social-network-tutorial-38ai
Demo: https://soroban-social-network.netlify.app/
Video: [link](https://www.youtube.com/watch?v=-RHDKI1Uk3g)

**Comment by briwylde08** [comment]
Heya! Thank you so much for your submission! We’ll review it and be in contact with you within the next week. 🎉

**Comment by ElliotFriend** [comment]
Hi there, @snowstorm134!

I really like your submission! It's really creative use of Soroban, and demonstrates great understanding of Soroban.

I'd only ask for one thing: Could you add some comments to your codebase? I think having the context right there in the `lib.rs` file can help take this example to the next level.

**Reply by snowstorm134** [reply]
Hi @ElliotFriend 
Thanks for your feedback and sorry for late response.

> I think having the context right there in the `lib.rs` file can help take this example to the next level.

Good idea
I will do it as soon as possible.

**Reply by ElliotFriend** [reply]
Sounds good! No problem at all. I was off for the weekend, too. No worries

**Reply by snowstorm134** [reply]
> Sounds good! No problem at all. I was off for the weekend, too. No worries

I just did it.
Can you please check?
Let me know if there is anything else I need to do.

**Reply by ElliotFriend** [reply]
Looks really good! Thanks for making those additions! Really cool project you put together!

**Comment by briwylde08** [comment]
Hello @snowstorm134! We've reviewed your submission! [🎉](https://emojipedia.org/party-popper/)

Given our guidelines, your submission may qualify for an award of $1,500 worth of XLM. To receive that award, please fill out [this form](https://stellarform.typeform.com/to/HBjEyCJt) completely and accurately.

Thanks again! Please let us know if you have any questions.

Best,
Bri

**Reply by snowstorm134** [reply]
Hi. 
Thanks ! I will do this in the next few days.

**Comment by briwylde08** [comment]
Thank you so much! This has been awarded per: https://github.com/stellar/sorobounty-spectacular/discussions/29.

Best,
Bri
